access-control-app/docs/api.md

```diff
@@ -582,7 +582,7 @@ Content-Type: application/json
 
 ### <a name="update-acl"></a> Update ACL
 
-An ACL can be updated by sending a PUT request to `%CMR-ENDPOINT%/acls/<concept-id>` where `concept-id` is the concept id of the ACL returned when it was created. `legacy-guid` field cannot be modified during update. The fields that uniquely identify an ACL cannot be updated, e.g. the `target` field of a system identity ACL. This limitation will eventually be removed as documented in CMR-3163.
+An ACL can be updated by sending a PUT request to `%CMR-ENDPOINT%/acls/<concept-id>`,along with a valid Launchpad token.  where `concept-id` is the concept id of the ACL returned when it was created. `legacy-guid` field cannot be modified during update. The fields that uniquely identify an ACL cannot be updated, e.g. the `target` field of a system identity ACL. This limitation will eventually be removed as documented in CMR-3163.
```

Comment:
a space after the comma please. One space after period as well (since we use proportional spaced fonts by default.

Reply:
Done.